scss/_base.scss

```diff
@@ -0,0 +1,23 @@
+html {
+  background: $htmlBG;
+}
+
+body {
+  min-height: 100vh;
+  max-width: 65em;
+  margin: 0 auto;
+  padding: 1em 2em;
+  color: $gray;
+
+  @media screen and (min-width: 40em) {
```

Comment:
48em appears to be a more consistent breakpoint, aligned with Bootstrap defaults. I will check to see if the spots that say 40em can be replaced with 48em.